src/plugins/intel_npu/src/backend/include/zero_init.hpp

```diff
@@ -37,6 +37,12 @@ class ZeroInitStructsHolder final {
     inline ze_context_handle_t getContext() const {
         return context;
     }
+    inline char* getGraphExtName() const {
+        return graphExtName;
+    }
+    inline ze_graph_dditable_ext_last_t* getGraphDDITableExt() {
```

Comment:
The ze_graph_dditable_ext_last_t is 1_5, but latest is 1.6, also need to change to support old diver